salt/docker/init.sls

```diff
@@ -16,6 +16,7 @@ include:
         Environment="NO_PROXY={{ salt['pillar.get']('proxy:no_proxy', '') }}"
   cmd.run:
     - name: systemctl daemon-reload
+    - stateful: True
```

Comment:
I believe the `file.managed` and `cmd.run` are different actions, so the `cmd.run` shouldn't stop the `file.managed` from being watched by other states, and shouldn't swallow those events itself. However I will double check that so I can confirm, thank you.

The idea behind the stateful here was to avoid it from appearing as `changed` during a highstate or orchestration, so we can have automated tests that run the orchestration twice and receive no changes on the second run.